crates/runtime/wit/deps/cli/exit.wit

```diff
@@ -1,4 +1,4 @@
 interface exit {
-  /// Exit the current instance and any linked instances.
+  /// Exit the curerent instance and any linked instances.
```

Comment:
Nit: Looks like a fat fingered key

Reply:
Indeed, but we should not fix a dependency